platform/eureka/controller.go

```diff
@@ -0,0 +1,72 @@
+package eureka
+
+import (
+	"time"
+
+	"reflect"
+
```

Comment:
kind of surprised this passed this goimports test, I believe "time" and "reflect" should be sorted alphabetically